mathics_django/web/templates/about.html

```diff
@@ -12,8 +12,10 @@
 			<img id="logo" class="load" src="{% static 'img/logo-heptatom.svg' %}" height="32" alt="Logo" />
 			<img id="logotext" class="load" src="{% static 'img/logo-text.svg' %}" height="26" alt="Mathics" />
 		</a>
+		<div class="menu">
+		  <a href="https://github.com/Mathics3" title="Go to Mathics3 github repository"><i class="fa fa-github"></i></a>
```

Comment:
github -> GitHub

Reply:
Ok - fixed in d574fa5. Onward and upward. (Docs in core need updating for the new navigation buttons).